sys/kern/tmpfs.c

```diff
@@ -628,33 +633,20 @@ static vnodeops_t tmpfs_vnodeops = {.v_lookup = tmpfs_vop_lookup,
                                     .v_mkdir = tmpfs_vop_mkdir,
                                     .v_rmdir = tmpfs_vop_rmdir,
                                     .v_access = vnode_access_generic,
+                                    .v_inactive = tmpfs_vop_inactive,
                                     .v_reclaim = tmpfs_vop_reclaim,
                                     .v_readlink = tmpfs_vop_readlink,
                                     .v_symlink = tmpfs_vop_symlink,
                                     .v_link = tmpfs_vop_link};
```

Comment:
Please add `,` after last struct member - then shitty clang-format will be happy and it will produce human readable output :)